Disable the `async-std` feature for `rfd` in `dioxus-desktop`

Currently, `dioxus-desktop` imports `rfd` with default features, which, by default enables `async-std`. If I import rfd with the `tokio` feature, it causes a compile error:
```toml
[dependencies]

dioxus = { version = "0.5.1", features = ["desktop"] }
rfd = { version = "0.14.0", default-features = false, features = ["xdg-portal", "tokio"]}
```

```
error[E0252]: the name `AsyncReadExt` is defined multiple times
  --> /home/soph/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ashpd-0.8.1/src/desktop/secret.rs:28:13
   |
26 | use futures_util::AsyncReadExt;
   |     -------------------------- previous import of the trait `AsyncReadExt` here
27 | #[cfg(feature = "tokio")]
28 | use tokio::{io::AsyncReadExt, net::UnixStream};
   |             ^^^^^^^^^^^^^^^^ `AsyncReadExt` reimported here
   |
   = note: `AsyncReadExt` must be defined only once in the type namespace of this module
help: you can use `as` to change the binding name of the import
   |
28 | use tokio::{io::AsyncReadExt as OtherAsyncReadExt, net::UnixStream};
   |             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0252]: the name `UnixStream` is defined multiple times
  --> /home/soph/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ashpd-0.8.1/src/desktop/secret.rs:28:31
   |
24 | use async_net::unix::UnixStream;
   |     --------------------------- previous import of the type `UnixStream` here
...
28 | use tokio::{io::AsyncReadExt, net::UnixStream};
   |                               ^^^^^^^^^^^^^^^ `UnixStream` reimported here
   |
   = note: `UnixStream` must be defined only once in the type namespace of this module
help: you can use `as` to change the binding name of the import
   |
28 | use tokio::{io::AsyncReadExt, net::UnixStream as OtherUnixStream};
   |                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error[E0252]: the name `File` is defined multiple times
 --> /home/soph/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ashpd-0.8.1/src/helpers.rs:6:13
  |
2 | use async_fs::File;
  |     -------------- previous import of the type `File` here
...
6 | use tokio::{fs::File, io::AsyncReadExt};
  |             ^^^^^^^^ `File` reimported here
  |
  = note: `File` must be defined only once in the type namespace of this module
help: you can use `as` to change the binding name of the import
  |
6 | use tokio::{fs::File as OtherFile, io::AsyncReadExt};
  |             ~~~~~~~~~~~~~~~~~~~~~

error[E0252]: the name `AsyncReadExt` is defined multiple times
 --> /home/soph/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ashpd-0.8.1/src/helpers.rs:6:23
  |
4 | use futures_util::AsyncReadExt;
  |     -------------------------- previous import of the trait `AsyncReadExt` here
5 | #[cfg(feature = "tokio")]
6 | use tokio::{fs::File, io::AsyncReadExt};
  |                       ^^^^^^^^^^^^^^^^ `AsyncReadExt` reimported here
  |
  = note: `AsyncReadExt` must be defined only once in the type namespace of this module
help: you can use `as` to change the binding name of the import
  |
6 | use tokio::{fs::File, io::AsyncReadExt as OtherAsyncReadExt};
  |                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

error: You can't enable both async-std & tokio features at once
  --> /home/soph/.cargo/registry/src/index.crates.io-6f17d22bba15001f/ashpd-0.8.1/src/lib.rs:10:1
   |
10 | compile_error!("You can't enable both async-std & tokio features at once");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0252`.
error: could not compile `ashpd` (lib) due to 5 previous errors
```

The compile error goes away if I remove the `desktop` feature, confirming that `dioxus-desktop` is to blame.

AFAIK dioxus doesn't use `async-std` anywhere (although it does use tokio), so the feature flag should be safe to remove.

Perhaps allow users to specify if they want to use rfd with async-std or tokio with a feature flag, like
```toml
[dependencies]
dioxus = { version = "0.5.1", features = ["desktop", "desktop-rfd-async-std"] }
```
or 
```toml
[dependencies]
dioxus = { version = "0.5.1", features = ["desktop", "desktop-rfd-tokio"] }
```

For now the workaround seems to be to import an earlier version of `rfd`, so that cargo considers them different crates.

--- a/packages/desktop/Cargo.toml
+++ b/packages/desktop/Cargo.toml
@@ -59,7 +59,7 @@ signal-hook = "0.3.17"
 
 [target.'cfg(any(target_os = "windows",target_os = "macos",target_os = "linux",target_os = "dragonfly", target_os = "freebsd", target_os = "netbsd", target_os = "openbsd"))'.dependencies]
 global-hotkey = "0.5.0"
-rfd = "0.14"
+rfd = { version = "0.14", default-features = false, features = ["xdg-portal"] }
 muda = "0.11.3"
 
 

--- a/packages/desktop/Cargo.toml
+++ b/packages/desktop/Cargo.toml
@@ -59,7 +59,7 @@ signal-hook = "0.3.17"
 
 [target.'cfg(any(target_os = "windows",target_os = "macos",target_os = "linux",target_os = "dragonfly", target_os = "freebsd", target_os = "netbsd", target_os = "openbsd"))'.dependencies]
 global-hotkey = "0.5.0"
-rfd = { version = "0.14", default-features = false, features = ["xdg-portal"] }
+rfd = { version = "0.14", default-features = false, features = ["xdg-portal", "tokio"] }
 muda = "0.11.3"
 
 

--- a/packages/html/Cargo.toml
+++ b/packages/html/Cargo.toml
@@ -26,7 +26,6 @@ enumset = "1.1.2"
 keyboard-types = { version = "0.7", default-features = false }
 async-trait = { version = "0.1.58", optional = true }
 tokio = { workspace = true, features = ["fs", "io-util"], optional = true }
-rfd = { version = "0.14", optional = true }
 futures-channel = { workspace = true }
 serde_json = { version = "1", optional = true }
 tracing.workspace = true

